src/kirby/components/button/button.component.scss

```diff
@@ -1,11 +1,65 @@
 @import '../../scss/web-imports';
 
 :host {
-  --kirby-button-border-radius: 2rem;
-  --kirby-button-background-color: #{get-color('primary')};
-  --kirby-button-border-color: #{get-color('primary')};
-  --kirby-button-color: #{get-color('primary-contrast')};
+  @mixin background-color($color) {
+    --kirby-button-background-color:#{get-color($color)};
```

Comment:
These mixins doesn't provide any additional value - they are just a 1-line wrapper for a 1-line def... Remove